addons/sky_3d/src/TimeOfDay.gd

```diff
@@ -279,6 +279,26 @@ func set_time(hour: int, minute: int, second: int) -> void:
 	set_total_hours(DateTimeUtil.hours_to_total_hours(hour, minute, second))
 
 
+func get_unix_timestamp() -> int:
+	var datetime_dict = {
+		"year": year,
+		"month": month,
+		"day": day,
+		"hour": floor(total_hours),
+		"minute": floor(fmod(total_hours, 1.0) * 60),
+		"second": floor(fmod(total_hours * 60, 1.0) * 60)
```

Comment:
Those 60s should be floats. Add a period to them. 60. Or 60.0. Either is fine, but make 1. match.